src/components/settings.tsx

```diff
@@ -177,7 +177,7 @@ class Settings extends React.Component<any, any> {
     const data: any[] = [];
     Object.keys(keyring)
       .sort((a, b) => { return a.toLowerCase() > b.toLowerCase() ? 1 : -1 })
-      .forEach((name) => { data.push({ name }) })
+      .forEach((name) => { data.push({ name, key: name }) })
```

Comment:
Having trouble seeing where `key` is used. Are you sure this is a safe update?

Reply:
It is used by React to handle rerenders of components in a list. It was throwing the error that it requires key on components. It's safe.